controllers/controller.js

```diff
@@ -11,3 +15,14 @@ exports.endpointsDescription = (req, res, next) => {
     res.status(200).send(JSON.parse(endpoints));
   });
 };
+
+exports.sendArticle = (req, res, next) => {
+  const { id } = req.params;
+  getArticleById(id)
+    .then((article) => {
+      res.status(200).send(article);
+    })
+    .catch((err) => {
+      res.status(err.status).send(err)
```

Comment:
here we should make use of next() and let a custom error handler send this response